### PR TITLE
I added my self to planet Xamarin

### PR DESCRIPTION
**Pull request by DamienDoumer**
:pencil2: **Blog Url**: https://doumer.me/

:scroll: **Feed Url**: https://doumer.me/feed/

By sending this pull request to add my blog I verify that I adhere to the Planet Xamarin blog guidelines (please check each item that applies):

- [x] I have a valid blog & RSS URL, both using HTTPS :lock: with a valid certificate
- [x] I host NO malicious or offensive content on the blog (including photos, swearing, etc.)
- [x] My blog is active with at least 3 Xamarin related blog posts in the last 6 months
- [x] In addition to the previous guideline; it should be apperant that the blog is active for a longer period of time and the posts on it are not fabricated in a small amount of time to comply to the '3 posts in last 6 months' guideline.
- [ ] I have applied a filter (Optional: if blog is only about Xamarin)
- [x] I have added my Author class to the csproj file
- [x] I have implemented `IAmACommunityMember` or one of the other derivatives
- [x] I understand, if I delete my blog, I will be deleted from Planet Xamarin
- [x] I understand, my blog may be removed at any time if any of these guidelines are broken.
- [x] The submitted blog is not owned by a company which sole purpose it is to advertise a product of productline.

Only the collaborators of this repository decide whether a blog is eligible or not. They will determine if the rules are followed and if the quality of the content is high enough. 

Opening another PR, after the first one is closed in a small period of time will not be reevaluated. If a PR is closed, the issues with your blog are too significant to be fixed within a matter of hours.

### CODE REVIEW

**Comment by Cheesebaron** [comment]
Fill out the PR template, please.

File: src/Firehose.Web/Authors/DamienDoumer.cs [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+            get { yield return new Uri("https://doumer.me/feed/"); }
+        }
+
+        public string TwitterHandle => "Damien_Doumer";
+
+        public string GravatarHash => "ecdd93df62c61daa04da17967f82d08d";
+        public string GitHubHandle => "damiendoumer";
+        public GeoPosition Position => new GeoPosition(4.07316844239285, 9.684240818616558);
+
+        public bool Filter(SyndicationItem item)
```
Comment (Cheesebaron) [inline src/Firehose.Web/Authors/DamienDoumer.cs]: Remove the filter. Default filter looks for Xamarin in title and categories.

**Comment by DamienDoumer** [comment]
Ok, thanks for pointing out the issue.
I just filled the form and made a new commit where everything is built and the filter is removed since it filters the Xamarin category of the posts by default.

**Comment by Cheesebaron** [comment]
Looks good to me now.

Welcome to PlanetXamarin.

**Comment by DamienDoumer** [comment]
ok cool. 👍

**Pull request merged**